### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Properties_ScreenReader: Same name property is provided for the "Alphabetical" buttons present in the toolbar.

### ISSUE THREAD

**Issue by Truptik12** [microsoft/azuredatastudio#20786]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #NVDA; #WCAG4.1.2; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Properties" button and hit ENTER key.
8. Tab through the toolbar and observe the announcement of the screen reader on the "Alphabetical" section.

### Actual:
Same name property is provided for the "Alphabetical" buttons present in the toolbar.

### Expected Result:
Different names should be provided for different buttons like "Alphabetical[A-Z]" and "Alphabetical[Z-A]" to avoid confusion.

### Attachment:
![Screen Recording (9)](https://user-images.githubusercontent.com/86710491/194825441-38fd9c6a-2d57-4f0a-b723-cd4745d3f81b.gif)

**Comment by aasimkhan30** [comment]
@Truptik12 , this issue is fixed in the latest insiders. Please retest and close this issue.

**Comment by Truptik12** [comment]
Verified the bug in the below environment 
Version: 1.40.0-insider (user setup)
Commit: d5cf088ba7b794859438fd2c1902249fd1e6a947
Date: 2022-10-26T05:45:47.459Z
VS Code: 1.67.0
Electron: 17.4.5
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.22621

The Screen reader is announcing "Alphabetical" and "Reverse Alphabetical" for the buttons. So the bug does not repro. Hence closing the bug. Please find the screenshot below.

![20786_Closed](https://user-images.githubusercontent.com/86710491/197971212-9d855a74-3c31-4080-b4a1-a0d5c4f1c9a2.png)
